doc/source/rest.j2

```diff
@@ -87,18 +101,45 @@ status code. It is possible to provide any number of |measures|.
 
 .. IMPORTANT::
 
-   While it is possible to send any number of (timestamp, value), it is still
-   needed to honor constraints defined by the |archive policy| used by the
+   While it is possible to send any number of (timestamp, value), they still
+   need to honor constraints defined by the |archive policy| used by the
    |metric|, such as the maximum |timespan|.
 
+Batch
+~~~~~
+
+It is also possible to batch |measures| sending, i.e. send several |measures|
+for different |metrics| in a simple call:
+
+{{ scenarios['post-measures-batch']['doc'] }}
+
+Or using named |metrics| of |resources|:
+
+{{ scenarios['post-measures-batch-named']['doc'] }}
+
+If some named |metrics| specified in the batch request do not exist, Gnocchi
+can try to create them as long as an |archive policy| rule matches:
+
+{{ scenarios['post-measures-batch-named-create']['doc'] }}
+
```

Comment:
Since that measures and aggregates have been distinguished now, maybe there should be a new section:
```
Aggregates
==========
```

Reply:
sorry, i don't understand your comment. i already have an aggregates section below?

Reply:
At some point before, the incoming data points and the processed(aggregated) ones are both called `measures`. But since one of my commit and the discussion with jd, the processed ones are now called `aggregates`. To explain in another way, now we have 'measure storage', 'aggregate storage', 'index storage', and the `Aggregates` I mentioned is the one in 'aggregate' storage.
To make the document more consistent, measures can only be pushed(POST), and the ones get read(GET) are actually `aggregates`. So I hope that the `Read` part here should belong to `Aggregates` section rather than `Measures` section.
BTW, I know that for the endpoint, `measures` is still used when getting aggregates, but I'm planning to deprecate that and create a new `aggregates` endpoint. #421

Reply:
so i understand the comment about terminology but i would probably have a discussion first about moving all GET actions to another endpoints. imo, it's a bad idea. people are use to CRUD concept and gnocch itself is already built on principle that everything is aggregated.

to me, aggregates endpoint is a custom-retrieval endpoint, not a simple read.

Reply:
To circle back on this discussion, I lean toward @Asu4ni about terminology because I encountered a lot of people confused about measures vs. aggregates and having it clearer is better. The changes proposed in #421 are orthogonal to this though – they could happen one day, but for now we should just focus on what exists. :)

That being said, that seems good enough for me now. The doc really talks about retrieving *aggregates* and not *measures*. :)